neo.UnitTests/UT_UIntBenchmarks.cs

```diff
@@ -15,7 +15,7 @@ namespace Neo.UnitTests
     [TestClass]
     public class UT_UIntBenchmarks
     {
-        int MAX_TESTS = 1000000; // 1 million
+        int MAX_TESTS = 1000; // 1 million
```

Comment:
Very good! It was too long for definitive tests.

Reply:
Can change the comment also to not say 1 million :-)